app/models/event.rb

```diff
@@ -147,7 +148,7 @@ def to_param
   # ActiveRecord :on_create
   def slugify!
     self.slug = "#{self.chapter.name} #{self.name.parameterize} #{self.id}".parameterize
```

Comment:
`self.id` is available only after DB persistence. It will not be available in `before_create`